npm/webpack-preprocessor/package.json

```diff
@@ -34,7 +34,6 @@
     "chalk": "3.0.0",
     "chokidar-cli": "2.1.0",
     "common-tags": "^1.8.2",
-    "cypress": "0.0.0-development",
```

Comment:
We should be able to keep this as `cypress` is still a dev dep

Reply:
yeah I updated the other PR to add this back in